desietc/util.py

```diff
@@ -63,6 +64,171 @@ def fit_spots(data, ivar, profile, area=1):
     return f, b, cov
 
 
+def shifted_profile(profile, dx, dy):
+    """Shift a reference profile with translations along the x and y axis.
+
+    All inputs are nominally 2D but can have other shapes as long as
+    they broadcast correctly. Input arrays with >2 dimensions are assumed
+    to have the pixels indexed along their last 2 dimensions.
+
+    Parameters
+    ----------
+    profile : array
+        Array of shape (...,ny,nx) with the spot profile(s) to use.
+    dx : array
+        Array of shape (...)
+        Position shift of the input profile to be applied in the x direction.
+    dy : array
+        Array of shape (...)
+        Position shift of the input profile to be applied in the y direction.
+
+    Returns
+    -------
+    array
+        Array (Shifted_profile) has shape (...,ny,nx) and is the profile whose position have been translated by dx
+        in the x direction and dy in the y direction.
+    """
+    nf = profile.shape[0]
```

Comment:
Yes indeed. I wrote the docstring to be similar to that of the "fit_spots" function. However assuming the shape of the input arrays was the most straightforward way of coding the new functions and consistent with the way the inputs are built in the sky.py code. I forgot to change the docstring accordingly. I've corrected that. 
I could, the other way around, let the docstring untouched and change the code accounting for possible change in the input shapes. I don't know if it is needed since the input constructed in the sky.py pipeline are always the same shape.